packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -282,6 +282,7 @@ class ListTile extends StatelessWidget {
     this.enabled = true,
     this.onTap,
     this.onLongPress,
+    this.onFocusChange,
```

Comment:
This seems like a sensible pass through. It should also be added to SwitchListTile, RadioListTile and CheckboxListTile.